Source/Lib/Codec/EbMdRateEstimation.c

```diff
@@ -266,10 +266,10 @@ EB_ERRORTYPE GetSaoOffsetsFractionBits(
             *fractionBitNum += 163840; //32768*5
 
             // code abs value of offsets
-            *fractionBitNum +=  mdRateEstimationArray->saoOffsetTrunUnaryBits[ABS(offset[0])];
-            *fractionBitNum +=  mdRateEstimationArray->saoOffsetTrunUnaryBits[ABS(offset[1])];
-            *fractionBitNum +=  mdRateEstimationArray->saoOffsetTrunUnaryBits[ABS(offset[2])];
-            *fractionBitNum +=  mdRateEstimationArray->saoOffsetTrunUnaryBits[ABS(offset[3])];
+            *fractionBitNum +=  mdRateEstimationArray->saoOffsetTrunUnaryBits[MIN(7, ABS(offset[0]))];
```

Comment:
Thanks for catching up this issue, @luofalei !
Could you modify 7 to (NUMBER_OF_SAO_OFFSET_TRUNUNARY_CASES - 1), or set the max input index with it at the beginning of GetSaoOffsetsFractionBits() when saoType != 0?

Reply:
Thanks for your suggestion. The commit was updated.